Frameworks/include/UIViewInternal.h

```diff
@@ -29,10 +29,33 @@
 #import "UWP/InteropBase.h"
 #import "UWP/WindowsUIXamlControls.h"
 
+#import "UIViewInternal.h"
```

Comment:
thanks; unintentional modification.

---
In reply to: [99915207](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r99915207) [](ancestors = 99915207)